CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (<https://github.com/openvinotoolkit/datumaro/pull/539>)
 - BraTS format (import-only) (.npy and .nii.gz), new `MultiframeImage`
   media type (<https://github.com/openvinotoolkit/datumaro/pull/628>)
+- Common Semantic Segmentation dataset format (import-only)
```

Comment:
Don't you think it's worth making not import-only? I feel, this format could be used extensively.

Reply:
This format is now needed only for import for WB.

Reply:
How WB is going to obtain such datasets? This format is crafted for OpenVINO and it is essential, that all our projects can work with it. Someone needs to be able to write in this format.

Reply:
I agree with this. We will need to support export in this format. But at the moment, the format is needed only for detection.
I think that the export task can wait.